xarray/backends/rasterio_.py

```diff
@@ -176,7 +176,7 @@ def open_rasterio(filename, parse_coordinates=None, chunks=None, cache=None, loc
 
         from affine import Affine
         da = xr.open_rasterio('path_to_file.tif')
-        transform = Affine.from_gdal(*da.attrs['transform'])
+        transform = Affine(*da.transform)
         nx, ny = da.sizes['x'], da.sizes['y']
         x, y = np.meshgrid(np.arange(nx)+0.5, np.arange(ny)+0.5) * transform
```

Comment:
if we convert all of this to doctests we can have the doctest CI check whether this still works. Would probably need to use `xarray.tutorial.open_rasterio` instead of `xarray.open_rasterio` to open an actual tif file, though.

Reply:
I'm pretty new to this, so not sure how to go about it. Is it simply a matter of adding >> before the example code and the expected output below?